+ source common/common_dnf.sh
++ source /root/mugen/libs/locallibs/common_lib.sh
+++ python3 --version
+++ '[' 0 -eq 0 ']'
+++ source /root/mugen/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start to prepare the test environment.'
+ message='Start to prepare the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to prepare the test environment.'
Thu May 25 17:06:48 2023 - INFO  - Start to prepare the test environment.
+ deploy_env
+ OLD_LANG=en_US.UTF-8
+ export LANG=en_US.UTF-8
+ LANG=en_US.UTF-8
++ grep enabled=
++ grep -nA4 name=OS /etc/yum.repos.d/openEuler.repo
++ awk -F - '{print $1}'
+ line1=
++ grep enabled=
++ grep -nA4 name=everything /etc/yum.repos.d/openEuler.repo
++ awk -F - '{print $1}'
+ line2=
+ LOG_INFO 'Finish preparing the test environment.'
+ message='Finish preparing the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Finish preparing the test environment.'
Thu May 25 17:06:50 2023 - INFO  - Finish preparing the test environment.
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start to run test.'
+ message='Start to run test.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to run test.'
Thu May 25 17:06:51 2023 - INFO  - Start to run test.
+ grep -B3 enabled=1 /etc/yum.repos.d/openEuler.repo
+ grep name=OS
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_dnf_enabled_enablerepo.sh line 32'
+ message='oe_test_dnf_enabled_enablerepo.sh line 32'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_dnf_enabled_enablerepo.sh line 32'
Thu May 25 17:06:52 2023 - ERROR - oe_test_dnf_enabled_enablerepo.sh line 32
+ '[' 0 -eq 1 ']'
+ return 0
+ dnf repolist
+ grep OS
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_dnf_enabled_enablerepo.sh line 34'
+ message='oe_test_dnf_enabled_enablerepo.sh line 34'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_dnf_enabled_enablerepo.sh line 34'
Thu May 25 17:06:59 2023 - ERROR - oe_test_dnf_enabled_enablerepo.sh line 34
+ '[' 0 -eq 1 ']'
+ return 0
+ grep -B3 enabled=1 /etc/yum.repos.d/openEuler.repo
+ grep name=everything
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_dnf_enabled_enablerepo.sh line 36'
+ message='oe_test_dnf_enabled_enablerepo.sh line 36'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_dnf_enabled_enablerepo.sh line 36'
Thu May 25 17:07:00 2023 - ERROR - oe_test_dnf_enabled_enablerepo.sh line 36
+ '[' 0 -eq 1 ']'
+ return 0
+ dnf repolist
+ grep everything
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_dnf_enabled_enablerepo.sh line 38'
+ message='oe_test_dnf_enabled_enablerepo.sh line 38'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_dnf_enabled_enablerepo.sh line 38'
Thu May 25 17:07:07 2023 - ERROR - oe_test_dnf_enabled_enablerepo.sh line 38
+ '[' 0 -eq 1 ']'
+ return 0
+ sed -i 'c enabled=0' /etc/yum.repos.d/openEuler.repo
+ sed -i 'c enabled=0' /etc/yum.repos.d/openEuler.repo
+ dnf repolist
+ grep 'OS\|everything'
Warning: failed loading '/etc/yum.repos.d/openEuler.repo', skipping.
No repositories available
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_dnf_enabled_enablerepo.sh line 42'
+ message='oe_test_dnf_enabled_enablerepo.sh line 42'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_dnf_enabled_enablerepo.sh line 42'
Thu May 25 17:07:14 2023 - ERROR - oe_test_dnf_enabled_enablerepo.sh line 42
+ '[' 0 -eq 1 ']'
+ return 0
+ dnf install -y sysstat
+ grep 'No match for argument: sysstat'
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_dnf_enabled_enablerepo.sh line 44'
+ message='oe_test_dnf_enabled_enablerepo.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_dnf_enabled_enablerepo.sh line 44'
Thu May 25 17:07:20 2023 - ERROR - oe_test_dnf_enabled_enablerepo.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ dnf --enablerepo=EPOL repolist
+ grep EPOL
Warning: failed loading '/etc/yum.repos.d/openEuler.repo', skipping.
Error: Unknown repo: 'EPOL'
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_dnf_enabled_enablerepo.sh line 46'
+ message='oe_test_dnf_enabled_enablerepo.sh line 46'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_dnf_enabled_enablerepo.sh line 46'
Thu May 25 17:07:28 2023 - ERROR - oe_test_dnf_enabled_enablerepo.sh line 46
+ '[' 0 -eq 1 ']'
+ return 0
+ LOG_INFO 'End of the test.'
+ message='End of the test.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End of the test.'
Thu May 25 17:07:29 2023 - INFO  - End of the test.
+ CASE_RESULT 0
+ case_re=0
+ test -z 7
+ test 7 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Thu May 25 17:07:30 2023 - ERROR - failed to execute the case.
+ exit 7
+ post_test
+ LOG_INFO 'Start to restore the test environment.'
+ message='Start to restore the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to restore the test environment.'
Thu May 25 17:07:31 2023 - INFO  - Start to restore the test environment.
+ clear_env
+ export LANG=en_US.UTF-8
+ LANG=en_US.UTF-8
+ sed -i 'c enabled=1' /etc/yum.repos.d/openEuler.repo
+ sed -i 'c enabled=1' /etc/yum.repos.d/openEuler.repo
+ LOG_INFO 'End of restore the test environment.'
+ message='End of restore the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End of restore the test environment.'
Thu May 25 17:07:33 2023 - INFO  - End of restore the test environment.
